nixos/modules/misc/ids.nix

```diff
@@ -356,6 +356,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      audiobookshelf = 327;
```

Comment:
No need for static uid, those are legacy.

Just create `users.users.audiobookshelf` without it.